packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -52,7 +52,7 @@ const mapToUriOriginFormatArrays = (value?: string[]) =>
 function ApplicationDetails() {
   const { id, guideId, tab } = useParams();
   const { navigate, match } = useTenantPathname();
-  const isGuideView = id && guideId && match(`/applications/${id}/guide/${guideId}`);
+  const isGuideView = !!id && !!guideId && match(`/applications/${id}/guide/${guideId}`);
```

Comment:
nit: use `Boolean()`?

Reply:
No, using `Boolean()` cannot rule out the possibility of `undefined` in the right hand side expression.

<img width="831" alt="image" src="https://github.com/logto-io/logto/assets/12833674/bc3c5180-2b47-4427-ba8f-6f3b93402c41">